Frontend/src/components/Barchart2.jsx

```diff
@@ -0,0 +1,305 @@
+import * as React from 'react';
```

Comment:
This methodology isn't proper. There is no need to create separate BarChart2,3,...n. It's just unnecessary redundancy.
Simple use <VerticalBarChart/> directly thrice. There is no point on creating three different files.